config/hardware/axis/Z/default_wiring_2M.cfg

```diff
@@ -0,0 +1,9 @@
+# In this file we include the first motor
+[include default_wiring_1M.cfg]
+
+# And then, add a 2nd
```

Comment:
"And then, add a 2nd motor."

Reply:
👍🏻 Thank you! I've updated the comment to be consistent with the other stepper files.